database/factories/TravelFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * A factory for Travel.
+ *
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Travel>
+ */
+class TravelFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string,string|int|\DateTime>
+     */
+    public function definition()
+    {
+        return [
+            'name' => $this->faker->word(),
+            'primary_contact_user_id' => User::all()->random()->id,
+            'destination' => $this->faker->word(),
+            'departure_date' => $this->faker->dateTimeBetween('now', '1 year'),
+            'return_date' => $this->faker->dateTimeBetween('now', '1 year'),
```

Comment:
There's no guarantee here that return date is after departure date.  Maybe adjust the time ranges for departure/return date so they can't overlap the wrong way.